new/LIT7091QeneAyte.xml

```diff
@@ -0,0 +1,89 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7091QeneAyte" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አይ<corr>ቴ</corr>፡ ናስምክ፡ ውሉድ፡ አመጻ፡ ዘተመዝበረ፡ ልብነ፡ በግብረ፡ አመጻ፡ ነፋስ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAyte nāsmǝk wǝlud ʾamaṣā za-tamazbara lǝbbǝna ba-gǝbra 
+                    ʾamaṣā nafās (Qǝne of the mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">Where shall we lay, o children, the iniquity that has destroyed our hearts 
+                    through the work(s) of iniquity, wind…</title>
```

Comment:
"through the iniquity of the spirit"

Reply:
I think they rather compare the works of iniquity with wind

Reply:
Yes, you should understand it as "the works/deeds of iniquity (which is) wind". Wind may be desctructive and takes away, esp. light things (in this case, we should also consider that "heart" means "intellect"). Maybe th translation can be a little different: Where shall we lay, children, the iniquity having destroyed our hearts though ..." But not necessarily. Nice qene. Maybe, consider also "lean upon" instead of "lay".